changelog/5115.bugfix.rst

```diff
@@ -0,0 +1 @@
+Do not treat warnings as errors during ``pytest_configure``. Before it led to an internal error
```

Comment:
```suggestion
Do not treat warnings as errors during ``pytest_configure``. Before it led to an internal error.
```

Reply:
INTERNALERROR happened due to the warning being treated as error in general (IIRC).